aries_vcx/src/indy/primitives/credential_definition.rs

```diff
@@ -208,7 +208,7 @@ impl CredentialDef {
 
     pub fn get_data_json(&self) -> VcxResult<String> {
         serde_json::to_string(&self)
-            .map_err(|err| VcxError::from_msg(VcxErrorKind::SerializationError, "Failed to serialize credential definition"))
+            .map_err(|_| VcxError::from_msg(VcxErrorKind::SerializationError, "Failed to serialize credential definition"))
```

Comment:
The entire implementation can be replaced by error-free
```
json!(&self).to_string()
```
which begs for question if this function need to actually exist 🤔

Reply:
This is out of scope of this PR